docs/developer/testing/ MULTI_CLOUD_TESTING_UPDATES.md

```diff
@@ -0,0 +1,135 @@
+# Multi Cloud Test Suite Updates for Backup / Restore cases 
+
+### Overview
+Running backup/restore test cases with cloud-provider specific snapshots enabled.
+
+The Test Suite now takes different env variables related to the cloud as flags and create the VSLs based on that cloud provider. Validating the backup / restore test cases on the same.
+
+### How multiple profiles are enabled to support Backup / Restore.
+In ideal cases, both the credentials / profile for BSL and VSL would be the same and usually we dont mention the separate creds for them, but this is different in CI environments. In CI environment, the cluster is provisioned in CI Cloud / Cluster which we dont have access to, hence supporting volume backup using our credentials which is mounted in CI environment is not a valid option. Hence we are using different methods to support these.
```

Comment:
s/In CI environment/In a CI environment/
s//In CI environment/In the prow CI environment/

Reply:
nit.. fix.." the cluster is provisioned in CI Cloud / Cluster"

Reply:
nit.. mounted in CI environment